memdx/ops_crud.go

```diff
@@ -33,6 +35,26 @@ func (o OpsCrud) encodeReqExtFrames(onBehalfOf string, buf []byte) (Magic, []byt
 			return 0, nil, err
 		}
 	}
+	if durabilityLevel > 0 {
+		if durabilityLevelTimeout == 0 {
+			buf, err = AppendExtFrame(ExtFrameCodeReqDurability, []byte{byte(durabilityLevel)}, buf)
+			if err != nil {
+				return 0, nil, err
+			}
+		} else {
+			durabilityTimeoutMillis := durabilityLevelTimeout / time.Millisecond
+			if durabilityTimeoutMillis > 65535 {
+				durabilityTimeoutMillis = 65535
```

Comment:
We should file a bug to make this an error instead in the future.

Reply:
GOCBC filed.